src/main/resources/assets/js/application/auth/tdprAuthModuleConfig.js

```diff
@@ -44,7 +39,21 @@ define(['angular'
                 }
 
             });
+        $httpProvider.interceptors.push(function($q,$injector, $location){
+              return {
+                  'responseError': function(response) {
+                      if (response.status === 401) {
+                          $location.path('/login');
+                          return $q.reject(response);
+                      } else {
```

Comment:
let's get rid of this else and the return in the if above - just add simple return at the end